_collections/_section_style_guide/cash.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cartrawler Cash
+position: 5
+type:
+description:
+right_code: >-
+---
+
+### CTCashTheme
+
+### Applying the Theme - Android
+Override the following properties in your SDK theme
+```xml
+   <style name="CarTrawlerSDKTheme" parent="CTDayNightTheme" >
+        //..
+        <item name="ctCashBgStartColor">@color/bannerBgStartColor</item>
+        <item name="ctCashBgEndColor">@color/bannerBgEndColor</item>
+        <item name="ctCashPrimaryTextColor">@color/colorOnSurface</item>
+        <item name="ctCashSecondaryTextColor">@color/colorSecondary</item>
+        <item name="ctCashLogoUrl">https://www.mylogo.com/mylogo.png</item>
+        <item name="ctCashSmallLogoUrl">https://www.mylogo.com/mylogo.png</item>
+        <item name="ctUSPLogoUrl">https://www.mylogo.com/mylogo.png</item>
+   </style>
+```   
+
+For every LogoUrl above you also have a way to provide a local drawable instead a remote url. All you need is to
```

Comment:
Just some grammar mistakes to fix.

"For every logo url you can provide a local drawable instead of a remote url."